example/test/widget_test.dart

```diff
@@ -8,23 +8,20 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
 
-import 'package:my_flutter_wrapper_app/main.dart';
+import 'package:didomi_sdk_example/main.dart';
```

Comment:
didomi_sample could be a better candidate?

Reply:
i'll look if i can change it with the tests MR